images/linux/scripts/installers/java-tools.sh

```diff
@@ -73,6 +83,7 @@ DocumentInstalledItemIndent "7 ($(/usr/lib/jvm/zulu-7-azure-amd64/bin/java -show
 DocumentInstalledItemIndent "8 ($(/usr/lib/jvm/zulu-8-azure-amd64/bin/java -showversion |& head -n 1)) (default)"
 DocumentInstalledItemIndent "11 ($(/usr/lib/jvm/zulu-11-azure-amd64/bin/java -showversion |& head -n 1))"
 DocumentInstalledItemIndent "12 ($(/usr/lib/jvm/zulu-12-azure-amd64/bin/java -showversion |& head -n 1))"
-DocumentInstalledItem "Ant ($(ant -version))"
+DocumentInstalledItem "Apache Ant ($(ant -version))"
 DocumentInstalledItem "Gradle ${gradleVersion}"
-DocumentInstalledItem "Maven ($(mvn -version | head -n 1))"
+DocumentInstalledItem "Apache Maven ($(mvn -version | head -n 1))"
+DocumentInstalledItem "Apache JMeter (5.2.1)"
```

Comment:
as for the version, something like `jmeter --version | grep -o "[0-9]\.[0-9]\.[0-9]"` should do the work and output only the Jmeter version (5.2.1 in this case)

Reply:
Done!